src/CodeCompliance/Domain/Checks/PrivateApi/MethodOverwritten/MethodIsOverwritten.php

```diff
@@ -38,7 +37,7 @@ public function getName(): string
      */
     public function getGuideline(): string
     {
-        return 'Please avoid usage of core method %s::%s() in the class %s';
+        return 'Please avoid overriding of core method %s::%s() in the class %s';
```

Comment:
`overriding` -> `overwriting`

Reply:
- [ ] Rename class name `MethodIsOverwritten` ->  `MethodIsOverridden`. 
- [ ] Rename rule name `PrivateApi:MethodIsOverwritten` -> `PrivateApi:MethodIsOverridden`
- [ ] Rename the rule in documentation

Reply:
Thanks, ++ 
The commit in documentation
https://github.com/spryker/spryker-docs/pull/1270/commits/9f4be4e62084ffa1ab02a1dd5731997c982ebcc7